docs/source/author/topics.rst

```diff
@@ -81,81 +78,130 @@ The following flow charts will take you step by step through the process of addi
+
+  <map name="learning-outcomes-map">
+    <area shape="rect" coords="240,100,317,135" href="#learning-outcomes-configuration-file">
+    <area shape="rect" coords="240,210,317,245" href="#application-structure-configuration-file">
+    <area shape="rect" coords="555,200,633,235" href="#learning-outcomes-configuration-file">
+    <area shape="rect" coords="240,330,317,362" href="../getting_started/helper_commands.html#update">
+  </map>
+  <img src="../_static/img/topics_adding_learning_outcomes_flowchart.png" usemap="#learning-outcomes-map">
 
 You will now be able to add learning outcomes to lessons and programming
 exercises by referencing the keys you specified in the learning outcomes configuration
 file.
 
-
-.. _adding-topics-content-curriculum-areas:
+.. _adding-curriculum-areas:
 
 Adding Curriculum Areas
 ------------------------------------------------------------------------------
 
-.. The following image can copied for be edits here:
-.. image:: ../_static/img/topics_adding_curriculum_areas_flowchart.png
+.. The following image can copied for be edits here: https://goo.gl/Vjv6XV
+.. The image is included as raw HTML because it has clickable nodes.
+.. raw:: html
+
+  <map name="curriculum-areas-map">
+    <area shape="rect" coords="240,100,317,135" href="#curriculum-areas-configuration-file">
+    <area shape="rect" coords="240,210,317,245" href="#application-structure-configuration-file">
+    <area shape="rect" coords="560,200,642,232" href="#curriculum-areas-configuration-file">
+    <area shape="rect" coords="240,330,317,362" href="../getting_started/helper_commands.html#update">
+  </map>
+  <img src="../_static/img/topics_adding_curriculum_areas_flowchart.png" usemap="#curriculum-areas-map">
 
 You will now be able to add curriculum areas to lessons and curriculum
 integrations by referencing the keys you specified in the curriculum areas
 configuration file.
 
-
-.. _adding-topics-content-programming-exercises:
+.. _adding-a-programming-exercise:
 
 Adding a Programming Exercise
 ------------------------------------------------------------------------------
 
-.. The following image can copied for be edits here:
-.. image:: ../_static/img/topics_adding_programming_exercises_flowchart.png
+.. The following image can copied for be edits here: https://goo.gl/Vjv6XV
+.. The image is included as raw HTML because it has clickable nodes.
+.. raw:: html
+
+  <map name="programming-exercises-map">
+    <area shape="rect" coords="554,470,633,505" href="#programming-exercises-configuration-file">
+    <area shape="rect" coords="240,572,317,605" href="#programming-exercises-configuration-file">
+    <area shape="rect" coords="240,690,317,725" href="#programming-exercises-structure-configuration-file">
+    <area shape="rect" coords="240,807,317,841" href="../getting_started/helper_commands.html#update">
+  </map>
+  <img src="../_static/img/topics_adding_programming_exercises_flowchart.png" usemap="#programming-exercises-map">
```

Comment:
this diagram missed the step to add the prog ex config file to the topic config file :(